frontend/packages/topology/src/components/export-app/__tests__/ExportViewLogButton.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { FlexItem } from '@patternfly/react-core';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import store from '@console/internal/redux';
+import ExportViewLogButton from '../ExportViewLogButton';
+
+type Component = typeof ExportViewLogButton;
+type Props = React.ComponentProps<Component>;
+let ViewLogButtonProps: Props;
+
+describe('ExportViewLogButton', () => {
+  beforeEach(() => {
+    ViewLogButtonProps = { namespace: 'test' };
+  });
+
+  it('Does not render a FlexBox', () => {
+    // mount required for using provider
+    const logButtonWrapper = mount(
+      <Provider store={store}>
+        <ExportViewLogButton {...ViewLogButtonProps} />
+      </Provider>,
+    );
+    expect(logButtonWrapper.find(FlexItem).exists()).toBe(false);
```

Comment:
Please check also that the button shows "View logs" and calls the onClick method when clicked.